python/paddle/fluid/dygraph/dygraph_to_static/convert_call_func.py

```diff
@@ -29,12 +29,21 @@
 
 from paddle.fluid.dygraph.dygraph_to_static import ProgramTranslator
 from paddle.fluid.dygraph.layers import Layer
+from paddle.fluid.dygraph.dygraph_to_static.convert_builtins_func import converted_len
 
 DECORATOR_NAMES = ['declarative', 'dygraph_to_static_func']
 program_translator = ProgramTranslator()
 to_static_func = program_translator.get_func
 
 
+def is_builtin_call(func):
```

Comment:
Generally, this is not good for readability. The "is_builtin_call" means "is_builtin_call" but "is_builtin_len", then the name "is_builtin_call" may confuse future developer.

I would suggest to delete "is_builtin_call" and put the logic "not is_builtin_len(func) and is_builtin(func)" where you use it to increase readability.

Reply:
Thanks，done.